src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java

```diff
@@ -586,6 +586,30 @@ public static int indexOfType(final Throwable throwable, final Class<? extends T
         return indexOf(throwable, type, fromIndex, true);
     }
 
+    /**
+     * Return whether the given throwable is a checked exception:
+     * that is, neither a RuntimeException nor an Error.
+     *
+     * <pre>
+     * ExceptionUtils.isCheckedException(new Exception())                    = true
+     * ExceptionUtils.isCheckedException(new SQLException())                 = true
+     * ExceptionUtils.isCheckedException(new RuntimeException())             = false
+     * ExceptionUtils.isCheckedException(new IllegalArgumentException(""))   = false
+     * ExceptionUtils.isCheckedException(new Throwable())                    = true
+     * ExceptionUtils.isCheckedException(null)                               = false
+     * </pre>
+     *
+     * @param throwable the throwable to check
+     * @return whether the throwable is a checked exception
+     * @see java.lang.Exception
+     * @see java.lang.RuntimeException
+     * @see java.lang.Error
+     * @since 3.12.1
+     */
+    public static boolean isCheckedException(final Throwable throwable) {
```

Comment:
The API name is misleading since it takes more than Exception as input. 
Going further, I think the type name in the method name is redundant and calling the API 'isChecked' should be enough.

Reply:
Changed